CMakeLists.txt

```diff
@@ -1010,8 +1010,10 @@ add_custom_target(examples
 # performance tests
 add_performance_test(element
   SOURCES test/performance/element.cpp
+  test/performance/priority.cpp
```

Comment:
Instead of adding these tests to an existing performance test, use the `add_performance_test` function to create a new one named `priority`.

Reply:
hi @goatshriek this one I could not figure out why its not working for me :( 

I've had the below change 

# performance tests
add_performance_test(element
  SOURCES test/performance/element.cpp
)

add_performance_test(priority
  SOURCES test/performance/priority.cpp
)

but when I run benchmark even after 'make clean' its not picking up the priority.cpp file. Could you please let me know what I'm missing here. Hence, as of now, added that priority.cpp along with the element tag

Reply:
I'm not sure why that would be, the usage above should be all that is needed. Sometimes I have seen issues where changing the `CMakeLists.txt` file causes problems in an existing build tree. Try making the change above and then doing the configure and build in a clean directory.

Reply:
ok done